docker-compose.yml

```diff
@@ -1,8 +1,8 @@
 version: '3'
 
 services:
-    static-pages:
-        build: client
+    # static-pages:
```

Comment:
@alv67 why did you remove service "static-pages"?
It was explicitly created to build the static website independently of the back-end.

I do not think it causes harm at all, on the other hand it simplifies things on CI/CD and for future integration of the https front-end - see #68

Reply:
You're right. I made some temporary modifications to do some tests. And pushed it to be able to transfer this WIP modification to my other working site. This was not intended to be pulled in this version. No problem at all to restore "static-pages" server.